### ISSUE TITLE
It crashes if a dependent key is a regular expression

### ISSUE THREAD

**Issue by josemarluedke** [ember-cli/eslint-plugin-ember#221]
Assuming one has something like this.

```js
import Component from '@ember/component';
import { match } from '@ember/object/computed';

export default Component.extend({
  postalCode: '',
  postalCodeIsValid: match('postalCode', /^\d{5}(-\d{4})?$/)
});
```

We get the following error:

```
The Broccoli Plugin: [SimpleConcatConcat] failed with:
TypeError: key.match is not a function
at dependentKeys.map (/somewhere/node_modules/eslint-plugin-ember/lib/utils/ember.js:414:36)
```

Using v5.0.3

**Comment by nickschot** [comment]
I'm also seeing this.

**Comment by rwjblue** [comment]
Would you mind putting together a failing test PR?

**Comment by btecu** [comment]
I'm also seeing this in the `5.2.0`.

**Comment by rwjblue** [comment]
I believe this is fixed in 5.3.0.
